static/images/about/README

```diff
@@ -0,0 +1,15 @@
+To edit the pictures, use GIMP
```

Comment:
Remind me to share some info about css border radius for this!

Reply:
already done! I have to change it in the README but now it is done with css